qir/qat/TestTools/Tests/Unit/BUILD

```diff
@@ -1,27 +1,25 @@
-load("@rules_cc//cc:defs.bzl", "cc_test")
-
-cc_test(
-    name = "unit",
-    srcs = glob(["*.cpp"]),
-    deps = [
-        "//qir/external:gmock",
-        "//qir/qat:AllocationManager",
-        "//qir/qat:Commandline",
-        "//qir/qat:Generators",
-        "//qir/qat:GroupingPass",
-        "//qir/qat:Llvm",
-        "//qir/qat:Logging",
-        "//qir/qat:ModuleLoader",
-        "//qir/qat:PostTransformValidation",
-        "//qir/qat:PreTransformTrimming",
-        "//qir/qat:Profile",
-        "//qir/qat:QatConfig",
-        "//qir/qat:QatTypes",
-        "//qir/qat:Rules",
-        "//qir/qat:StaticResourceComponent",
-        "//qir/qat:TestTools",
-        "//qir/qat:TransformationRulesPass",
-        "//qir/qat:ValidationPass",
-        "//qir/qat:Validator",
-    ],
-)
+# cc_test(
```

Comment:
Why keep this file if it's fully commented out? Is it going to be used later?

Reply:
Thanks for picking up on this. I forgot about this one, but obviously it should not be commented out.